spec/unit/webrtc/groupCall.spec.ts

```diff
@@ -280,6 +289,143 @@ describe('Group Call', function() {
             }
         });
 
+        describe("call feeds changing", () => {
+            let call: MockCall;
+            const currentFeed = new MockCallFeed(FAKE_USER_ID_1);
+            const newFeed = new MockCallFeed(FAKE_USER_ID_1);
+
+            beforeEach(async () => {
+                jest.spyOn(groupCall, "getUserMediaFeedByUserId");
+                // @ts-ignore Mock
+                jest.spyOn(groupCall, "addUserMediaFeed");
+                // @ts-ignore Mock
+                jest.spyOn(groupCall, "replaceUserMediaFeed");
+                // @ts-ignore Mock
+                jest.spyOn(groupCall, "removeUserMediaFeed");
+                // @ts-ignore Mock
+                jest.spyOn(groupCall, "addScreenshareFeed");
+                // @ts-ignore Mock
+                jest.spyOn(groupCall, "replaceScreenshareFeed");
+                // @ts-ignore Mock
+                jest.spyOn(groupCall, "removeScreenshareFeed");
+
+                jest.spyOn(currentFeed, "dispose");
+                jest.spyOn(newFeed, "measureVolumeActivity");
+
+                jest.spyOn(groupCall, "emit");
+
+                call = new MockCall(room.roomId, groupCall.groupCallId);
+
+                await groupCall.create();
+            });
+
+            it("ignores changes, if we can't get user id of opponent", async () => {
+                const call = new MockCall(room.roomId, groupCall.groupCallId);
+                call.getOpponentMember = (() => {}) as unknown as any;
+
+                // @ts-ignore Mock
+                expect(() => groupCall.onCallFeedsChanged(call)).toThrowError();
+            });
+
+            describe("usermedia feeds", () => {
+                it("adds new usermedia feed", async () => {
+                    // @ts-ignore Mock
+                    call.remoteUsermediaFeed = newFeed;
+                    // @ts-ignore Mock
+                    groupCall.onCallFeedsChanged(call);
+
+                    // @ts-ignore Mock
+                    expect(groupCall.addUserMediaFeed).toHaveBeenCalledWith(newFeed);
+                });
+
+                it("replaces usermedia feed", async () => {
+                    groupCall.userMediaFeeds = [currentFeed.typed()];
+
+                    // @ts-ignore Mock
+                    call.remoteUsermediaFeed = newFeed;
```

Comment:
If you're doing this repeatedly, I would recommend adding the `remoteUsermediaFeed` field to `MockedCall` so you don't have to bypass TypeScript every time